utils/outputs/meta.go

```diff
@@ -0,0 +1,39 @@
+package outputs
```

Comment:
(1) My gut feeling is that `utils/outputs/meta.go` isn't the best place to start stuffing HTTP responses. Perhaps it should be under the controller like `controller/responses/`?
(2) I think this file should be broken up.

Reply:
utils/outputs was meant for outputs to http controllers.  I can rename if it does not make sense.

Reply:
resolved: https://github.com/brave-intl/bat-go/pull/820/commits/2f953cdeb8596dd694927d6d6006da49284aa038